RecoLocalCalo/HGCalRecProducers/plugins/HGCalCLUEAlgo.cc

```diff
@@ -214,6 +214,9 @@ math::XYZPoint HGCalCLUEAlgoT<T>::calculatePosition(const std::vector<int>& v, c
     float x_log = 0.f;
     float y_log = 0.f;
     for (auto i : v) {
+      //for silicon only just use 1+6 cells = 1.3cm for all thicknesses
+      if (distance(i, maxEnergyIndex, layerId, false) > positionDeltaRho_)
```

Comment:
Maybe we could use the `distance2` to avoid `sqrt`?

Reply:
done, thanks